### PR TITLE
Add node 14 and updated node latest

### PR DESCRIPTION
**Pull request by david-valero-mrjeff**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -41,7 +41,10 @@ build-node: build-base
 
 	docker build --no-cache -t "mrjeffapp/jenkins-pipeline-node:12-${COMMIT}" --build-arg base=${COMMIT} --build-arg node_version=12 -f node/Dockerfile .
 	docker tag "mrjeffapp/jenkins-pipeline-node:12-${COMMIT}" 'mrjeffapp/jenkins-pipeline-node:12'
-	docker tag 'mrjeffapp/jenkins-pipeline-node:12' 'mrjeffapp/jenkins-pipeline-node:latest'
+
+	docker build --no-cache -t "mrjeffapp/jenkins-pipeline-node:14-${COMMIT}" --build-arg base=${COMMIT} --build-arg node_version=14 -f node/Dockerfile .
+    docker tag "mrjeffapp/jenkins-pipeline-node:14-${COMMIT}" 'mrjeffapp/jenkins-pipeline-node:14'
```
Comment (hectorespert) [inline Makefile]: Check indentation

**Review by hectorespert — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ Based in the [CircleCI images project](https://github.com/circleci/circleci-imag
 - `mrjeffapp/jenkins-pipeline-node:8`
 - `mrjeffapp/jenkins-pipeline-node:10`
 - `mrjeffapp/jenkins-pipeline-node:12`
+- `mrjeffapp/jenkins-pipeline-node:14`
```
Comment (hectorespert) [inline README.md:23]: 👍

**Review by hectorespert — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (david-valero-mrjeff) [reply Makefile]: Copy & paste from the other lines fixed the issue

**Pull request merged**